### PR TITLE
docs: Update faq.en-US.md

### PR DESCRIPTION
**Pull request by scouttyg**
Updating a bit of wording and making flow a bit more clear

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered docs/react/faq.en-US.md](https://github.com/scouttyg/ant-design/blob/patch-6/docs/react/faq.en-US.md)

### CODE REVIEW

**Review by abenhamdine — APPROVED** [review]

File: docs/react/faq.en-US.md [hunk 1]
```diff
@@ -7,49 +7,51 @@ Here are the frequently asked questions about Ant Design and antd that you shoul
 
 ---
 
-### Are you going to provide Sass/Stylus(etc.) style file?
+### Will you provide Sass/Stylus(etc.) style files in addition to the Less style files currently included?
 
-No plan, but you can convert Less to Sass/Stylus(etc.) with tools that you can find on Google.
+There is currently no plan to add support for Sass/Stylus style files, but using tools on Google you can easily convert the provided Less files to your desired style format.
```
Comment (07akioni) [inline docs/react/faq.en-US.md:12]: keep (etc.)?

File: docs/react/faq.en-US.md [hunk 2]
```diff
@@ -7,49 +7,51 @@ Here are the frequently asked questions about Ant Design and antd that you shoul
...
-This has been fixed since `3.11.x`. If you're using an older version, you can use `<Select getPopupContainer={trigger => trigger.parentElement}>` to render a component inside Popover. (Or other `getXxxxContainer` props)
+This is an old bug that has been fixed since `v3.11.x`. If you're using an older version, you can use `<Select getPopupContainer={trigger => trigger.parentElement}>` to render a component inside Popover. (Or other `getXxxxContainer` props)
 
 https://ant.design/components/select/#Select-props
 
-related issue: [#3487](https://github.com/ant-design/ant-design/issues/3487) [#3438](https://github.com/ant-design/ant-design/issues/3438)
+Related issue: [#3487](https://github.com/ant-design/ant-design/issues/3487) [#3438](https://github.com/ant-design/ant-design/issues/3438)
 
-### `Select Dropdown DatePicker TimePicker Popover Popconfirm` scrolls with the page?
+### How do you prevent `Select Dropdown DatePicker TimePicker Popover Popconfirm` scrolling with the page?
```
Comment (07akioni) [inline docs/react/faq.en-US.md:22]: The question is ask by a user. I think `How do I` is better than `How do you`.

File: docs/react/faq.en-US.md [hunk 3]
```diff
@@ -7,49 +7,51 @@ Here are the frequently asked questions about Ant Design and antd that you shoul
...
+### Will you provide other themes?
 
-No, we follow Ant Design specification. https://github.com/ant-design/ant-design/issues/1241
+No, we follow Ant Design's design specification. 
 
-### How to modify `Menu`/`Button`(etc.)'s style?
+Related issue: https://github.com/ant-design/ant-design/issues/1241
 
-You can override its style but we don't recommend doing so. antd is not only a set of React components but also a design specification.
+### How do you modify `Menu`/`Button`(etc.)'s style?
```
Comment (07akioni) [inline docs/react/faq.en-US.md:38]: the same

File: docs/react/faq.en-US.md [hunk 4]
```diff
@@ -7,49 +7,51 @@ Here are the frequently asked questions about Ant Design and antd that you shoul
...
 
-### How to replace Moment.js to Day.js to reduce bundle size？
+While you can override a component's style, we don't recommend doing so. antd is not only a set of React components, but also a design specification as well.
+
+### How do you replace Moment.js with Day.js to reduce bundle size？
```
Comment (07akioni) [inline docs/react/faq.en-US.md:42]: the same

File: docs/react/faq.en-US.md [hunk 5]
```diff
@@ -59,56 +61,56 @@ Try [Space](https://ant.design/components/space/) component to make them aligned
...
 
-Yes, antd is designed to develop a complete background application, we override some global styles for styling convenience, and it can't be removed now. More info at https://github.com/ant-design/ant-design/issues/4331 .
+Yes, antd is designed to help you develop a complete background application. To do so, we override some global styles for styling convenience, and currently these cannot be removed or changed. More info at https://github.com/ant-design/ant-design/issues/4331 .
 
 Alternatively, follow the instructions in [How to avoid modifying global styles?](docs/react/customize-theme#How-to-avoid-modifying-global-styles-?)
 
 ### I cannot install `antd` and `antd`'s dependencies in mainland China.
 
-Long live the Girl Friend Wall! And try [cnpm](http://npm.taobao.org/).
+Long live the Great Firewall! To potentially solve this, try [cnpm](http://npm.taobao.org/).
```
Comment (07akioni) [inline docs/react/faq.en-US.md:70]: Hahahaha, there is no Great Firewall. Just keep it.

Comment (07akioni) [reply docs/react/faq.en-US.md:70]: or just remove it. keep `To potentially solve this, try [cnpm](http://npm.taobao.org/).` to sync with chinese verison doc.

**Comment by scouttyg** [comment]
@07akioni updated the file with your suggested changes! Take another look and tell me what you think 😄

File: docs/react/faq.en-US.md [hunk 6]
```diff
@@ -59,56 +61,56 @@ Try [Space](https://ant.design/components/space/) component to make them aligned
...
 
 If you need some features which should not be included in antd, try to extend antd's component with [HOC](https://gist.github.com/sebmarkbage/ef0bf1f338a7182b6775). [more](https://medium.com/@dan_abramov/mixins-are-dead-long-live-higher-order-components-94a0d2f9e750#.eeu8q01s1)
 
-### How do I fix dynamic style when open Content Security Policy (CSP)?
+### How do I fix dynamic styles while using a Content Security Policy (CSP)?
 
 You can configure `nonce` by [ConfigProvider](/components/config-provider/#Content-Security-Policy).
 
-### When I set `mode` to DatePicker/RangePicker, I cannot select year or month anymore?
+### When I set `mode` to DatePicker/RangePicker, why can I not select a year or month anymore?
```
Comment (07akioni) [inline docs/react/faq.en-US.md:102]: should it be `why can't I`?

File: docs/react/faq.en-US.md [hunk 7]
```diff
@@ -59,56 +61,56 @@ Try [Space](https://ant.design/components/space/) component to make them aligned
...
+Like [the explaination](https://github.com/ant-design/ant-design/issues/11586#issuecomment-429189877) explains, this is because `<DatePicker mode="year" />` does not equal the `YearPicker`, nor is `<RangePicker mode="month" />` equal to `MonthRangePicker`. The `mode` property was added to support [showing time picker panel in DatePicker](https://github.com/ant-design/ant-design/issues/5190) in antd 3.0, which simply controls the displayed panel, and won't change the original date picking behavior of `DatePicker/RangePicker` (for instance you will still need to click date cell to finish selection in a DatePicker, whatever the `mode` is).
```
Comment (07akioni) [inline docs/react/faq.en-US.md:109]: `DatePicker/RangePicker` -> `DatePicker`/`RangePicker`

File: docs/react/faq.en-US.md [hunk 8]
```diff
@@ -59,56 +61,56 @@ Try [Space](https://ant.design/components/space/) component to make them aligned
...
 
 ##### Workaround
 
-You can refer to [this article](https://juejin.im/post/5cf65c366fb9a07eca6968f9) or [this article](https://www.cnblogs.com/zyl-Tara/p/10197177.html), using `mode` and `onPanelChange` to encapsulate a `YearPicker` or `MonthRangePicker` for your needs. Or you can wait for our [antd@4.0](https://github.com/ant-design/ant-design/issues/16911), in which we are planing (already) to [add more XxxPickers](https://github.com/ant-design/ant-design/issues/4524#issuecomment-480576884) for those requirments.
+You can refer to [this article](https://juejin.im/post/5cf65c366fb9a07eca6968f9) or [this article](https://www.cnblogs.com/zyl-Tara/p/10197177.html), using `mode` and `onPanelChange` to encapsulate a `YearPicker` or `MonthRangePicker` for your needs. Or you can wait for our [antd@4.0](https://github.com/ant-design/ant-design/issues/16911), in which we are already planning to [add more XxxPickers](https://github.com/ant-design/ant-design/issues/4524#issuecomment-480576884) to meet those requirments.
```
Comment (07akioni) [inline docs/react/faq.en-US.md:113]: requirments -> requirements

Comment (scouttyg) [reply docs/react/faq.en-US.md:113]: @07akioni Nice catch, I'll change that

File: docs/react/faq.en-US.md (hunk 6 above)
Comment (scouttyg) [reply docs/react/faq.en-US.md:102]: @07akioni In this case either would work -- in my mind `not` was connected more to `select` (e.g. `not select`) vs being connected to myself (`why can not I`).

File: docs/react/faq.en-US.md (hunk 8 above)
Comment (07akioni) [reply docs/react/faq.en-US.md:113]: Many thanks to you. We are really using plastic English before. Hahaahhahahaha

**Review by 07akioni — APPROVED** [review]

**Review by xrkffgg — APPROVED** [review]

**Pull request merged**